pkg/metrics/exporter/snapshot/metrics.go

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021. Alibaba Cloud. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package snapshot
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	daemonIDLabel = "daemon_id"
+	timeLabel     = "time"
+	eventLabel    = "event"
+)
+
+var (
+	NydusdEvent = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "nydusd_lifetime_event_times",
+			Help: "nydusd lifetime event times.",
```

Comment:
How about renaming it to `nydusd_lifetime_events`

Reply:
Please follow Prom name suggestions https://prometheus.io/docs/instrumenting/writing_exporters/#naming

Reply:
Ok, thanks.